<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqPlotMatrixDisplayPanel</class>
 <widget class="QWidget" name="pqPlotMatrixDisplayPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="pqTreeView" name="Series">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="ActivePlotColorLabel">
       <property name="text">
        <string>Active Plot Color:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="pqColorChooserButton" name="ActivePlotColor" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ScatterPlotsColorLabel">
       <property name="text">
        <string>Scatter Plots Color:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="pqColorChooserButton" name="ScatterPlotsColor" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="HistogramColorLabel">
       <property name="text">
        <string>Histogram Color:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="pqColorChooserButton" name="HistogramColor" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ActivePlotMarkerSizeLabel">
       <property name="text">
        <string>Active Plot Marker Size:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="ActivePlotMarkerSize"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="ScatterPlotMarkerSizeLabel">
       <property name="text">
        <string>Scatter Plot Marker Size:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="ScatterPlotMarkerSize"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="ActivePlotMarkerStyleLabel">
       <property name="text">
        <string>Active Plot Marker Style:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="ActivePlotMarkerStyle">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cross</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Circle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamond</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="ScatterPlotMarkerStyleLabel">
       <property name="text">
        <string>Scatter Plot Marker Style:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="ScatterPlotMarkerStyle">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cross</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Circle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamond</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqTreeView</class>
   <extends>QTreeView</extends>
   <header>pqTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>pqColorChooserButton</class>
   <extends>QWidget</extends>
   <header>pqColorChooserButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
